### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by cfergeau**
Mention cert renewal process in the docs, and mention odo too.

### CODE REVIEW

File: docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc [hunk 1]
```diff
@@ -1,12 +1,16 @@
 [id="troubleshooting-expired-certificates_{context}"]
 = Troubleshooting expired certificates
 
-Each released `{bin}` binary includes an embedded system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
+Prior to {rh-prod} 1.2.0, each released `{bin}` binary included a system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
```
Comment (robin-owen) [inline docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: Don't we still include the embedded system bundle?

File: docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 Using an older `{bin}` binary or system bundle will result in an expired certificates error.
 
+Starting from {prod} 1.2.0, the embedded certificates can be automatically renewed by {bin}.
+The certificate renewal process is triggered when needed when [command]`{bin} start` is run.
+When certificates need to be renewed, this can adds up to 5 minutes to the cluster startup time.
```
Comment (robin-owen) [inline docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: We try to avoid phrasal verbs (for example, "startup"). Reword to "this can add up to five minutes to the start time of the cluster"?

File: docs/source/getting-started/content/proc_accessing-the-openshift-cluster.adoc [hunk 3]
```diff
@@ -89,5 +89,6 @@ The `monitoring` cluster operator is expected to report `Unknown` availability.
 ====
 +
   . The link:https://docs.openshift.com/container-platform/latest/applications/projects/working-with-projects.html[OpenShift documentation] covers the creation of projects and applications.
+  link:https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/understanding-odo.html[OpenShift Do] (`odo`) can also be used to create OpenShift projects and applications from the command line.
```
Comment (robin-owen) [inline docs/source/getting-started/content/proc_accessing-the-openshift-cluster.adoc]: :+1:

File: docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc (hunk 1 above)
Comment (cfergeau) [reply docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: 'embedded' is repeated twice in the same sentence, which is not so nice. I don't think the 'embedded bundle' VS 'bundle' is really important here.

Comment (robin-owen) [reply docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: Sure, that makes sense. But the inclusion of "prior to {rh-prod} 1.2.0" implies that since 1.2.0, there is no included system bundle. Does that make sense?

Comment (cfergeau) [reply docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: Oooh ok, I thought you were ascking about the removal of 'embedded'
Rephrase it to "Prior to {rh-prod} 1.2.0, the system bundle included in each released `{bin}` binary expired 30 days after the release due to certificates embedded in the OpenShift cluster." ?

Comment (robin-owen) [reply docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc]: Sounds good to me. :+1:
